bundle/Sail-Operator-ReadMe.md

```diff
@@ -0,0 +1,54 @@
+# About the Sail Operator
+
+The present implementation is based on the open source Istio project. The implementation pulls its code from the upstream Istio master with no changes to the codebase.
+
+# Prerequisites
+
+You have deployed a cluster on OpenShift Container Platform 4.13 or later.
+
+# Installing the Sail Operator
+
+1. Create the project where Istio is going to be deployed:  
+
+``` 
+$ oc new-project istio-system
+```
+
+2. And give it privileges (workaround while anyuid upstream PR is not merged)  
+
+```
+$ oc adm policy add-scc-to-group anyuid system:serviceaccounts:istio-system
+$ oc adm policy add-scc-to-group privileged system:serviceaccounts:istio-system
+```
+
+3. Navigate to the operator hub 
+
+4. Click Operator >> Operator Hub
+
+5. Search on Service Mesh. 
+
+6. Locate the community version of the Service Mesh operator. (It’s called maistraoperator now, but it will change in the future.)
+
+7. Click the community operator.
+
+8. At the prompt that you’re using the community operator click **Continue**.
+
+9. In the Operator subscription window, verify it’s version 3.0, and click **Install**. 
+
+10. Use the defaults during the installation process, and click **Install** to continue.
+
+11. in the OpenShift console, select ```istio-system``` in the **Project** drop-down menu. 
+
+12. Click Operators >> Installed Operators to confirm the Operator is installed
+
+13. Click the maistraoperator.
+
+14. Click **Istio Helm Install**.
+
+15. Click **Create Istio Helm Install**.
+
+16. Accept the defaults and click **Create**. This creates the control plane.
+
+17. Click Workloads > Pods  
+
+Verify the pods were created.
```

Comment:
After this point you could say something like:

Istio is installed and ready to be used. Follow the documentation [link to istio.io] for usage.

Reply:
@jwendell I've updated the PR to capture all the suggestions. I also tried to add a little information indicating how to verify the task was successfully completed. Please take a look at line 8 in each task. We might want to tweak them a little, particularly for verifying the pods are up and running. Thanks!